### ISSUE TITLE
Provide mechanism(s) to ensure the "best" result is first in the apps page

### ISSUE THREAD

**Issue by mfisher87** [regolith-linux/ilia#26]
When using `ilia` as my application launcher, it doesn't consistently put the result I want first like my preferred `rofi` configuration does, and there isn't currently a way to configure sort or matching behavior.

Out of the box behavior from `ilia` (`ilia -p apps`):

https://user-images.githubusercontent.com/3608264/171771814-a86a143b-acda-4906-a1a9-011312266686.mp4

The two `rofi` configurations that I feel consistently provide good matches are:

* `rofi -show drun -drun-match-fields name`: _only_ search on the application name (note that there are few results) 

https://user-images.githubusercontent.com/3608264/171772032-421bc212-9ff9-4e2c-85df-69ccbcdabca7.mp4

* `rofi -show drun -sort -sorting-method fzf`: search on all application metadata and provide the "best" results first... maybe this is calculated by the number of matches in the application metadata (e.g. application name + application description matches is better than just an application description match)? _waves hands_

https://user-images.githubusercontent.com/3608264/171772011-65327aa7-08cf-4271-99be-dfa8276f8f07.mp4

**Comment by mfisher87** [comment]
Either GitHub corrupted the video files on upload or there's an issue with my browser. I'll try and convert them to something more web-friendly.

**Comment by mfisher87** [comment]
Videos should be working now. Took me 3 or 4 tries of different codecs before it finally worked. `libx264` codec seems to have been the trick for some reason. Or maybe it was one of the other flags I tried more or less randomly. :shrug:

**Comment by kgilmer** [comment]
Wow, thanks for the videos, very helpful @mfisher87 .  While the feature you're requesting could def be added, FYI one thing happening under the hood in `ilia` is that the last 32 app selections are saved in a list.  When `ilia` starts, it pulls those top 32 up in the ranking by frequency.  So, over time your most commonly used apps should rise to the top.  You can see what's being stored by ilia in gsettings via `gsettings get org.regolith-linux.ilia app-launch-counts`

**Comment by anamyk** [comment]
Same frustration for me here. I really like the Regolith project, using it since several years. Just upgraded from Regolith 1 to Regolith 3 - and i need my sorting options for the launcher. @mfisher87 Did you find a solution? Do you still use rofi?

**Comment by mfisher87** [comment]
Yes, I'm still using rofi. I've been too busy to get on to Regolith 3 so far, I'm ashamed to say!

https://github.com/mfisher87/provision-desktop/blob/b0e02bf6b6ddee94f3c0e4c0cad8e86692245159/files/regolith2-cfg/i3/config.d/config#L132-L134
